frontend/src/component/project/Project/Import/configure/ConfigurationStage.tsx

```diff
@@ -38,129 +37,133 @@ const MaxSizeMessage = styled(Typography)(({ theme }) => ({
     color: theme.palette.text.secondary,
 }));
 
-type ImportMode = 'file' | 'code';
+export type ImportMode = 'file' | 'code';
 
-const isValidJSON = (json: string) => {
-    try {
-        JSON.parse(json);
-        return true;
-    } catch (e) {
-        return false;
-    }
-};
-
-interface IConfigurationSettings {
-    environment: string;
-    payload: string;
-}
+export const ConfigurationTabs: FC<{
```

Comment:
configuration stage exposes small components. They can be composed in the parent who owns the state